src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPCompletionProposal.java

```diff
@@ -65,6 +73,12 @@ public class LSPCompletionProposal extends LookupElement implements Pointer<LSPC
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LSPCompletionProposal.class);
 
+    private static final Pattern TEMPLATE_VARIABLE_PATTERN = Pattern.compile("\\$(?:\\{\\d:.+?}|\\d+)");
```

Comment:
As stated in the comment, these patterns should work well for languages where invocations take the form of parenthesized, comma-delimited argument lists. If supported languages with other invocation syntaxes are found, it should be pretty simple to extend the pattern to support them as well, but until such concrete examples are found, this should be a lightweight way to help determine that a code snippet is explicitly for an invocation argument list.